manifests/default_mods.pp

```diff
@@ -12,7 +12,7 @@
       if versioncmp($apache_version, '2.4') >= 0 {
         # Lets fork it
         # Do not try to load mod_systemd on RHEL/CentOS 6 SCL.
-        if ( !($::osfamily == 'redhat' and versioncmp($::operatingsystemrelease, '7.0') == -1) and !($::operatingsystem == 'Amazon' and versioncmp($::operatingsystemrelease, '2014.09') <= 0  ) ) {
+        if ( !($::osfamily == 'redhat' and versioncmp($::operatingsystemrelease, '7.0') == -1) and !$::operatingsystem == 'Amazon' ) {
```

Comment:
The original implementation seems to imply that Amazon Linux after 2014.09 uses systemd. Is this incorrect?